src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -176,7 +176,19 @@ def track_initial_segment_event(
         first_replay_received.send_robust(project=project, sender=Project)
 
     # Replay videos are not billed for now.
```

Comment:
comment is outdated?

Reply:
No this outcome is not billed.  We're adding it so the PMs can access the data in looker.

Reply:
additionally: should there be a test covering this change?